core/modules/css/testCssAccess.cc

```diff
@@ -564,7 +564,7 @@ BOOST_AUTO_TEST_CASE(testCreateMatchTable) {
 
     BOOST_CHECK_THROW(createMatchTable("dbA", "MatchTable", "(INT I)", mParams), TableExists);
 
-    MatchTableParams params1{"dirTable1", "dirCol1", "dirTable2", "dirCol2", "flagCol"};
```

Comment:
can anything be done to improve the magic number here? a comment, or named var, or?